docs/guidance/aws-eks-gpu-cluster.md

```diff
@@ -0,0 +1,74 @@
+# Start Amazon EKS Cluster with GPUs for KubeRay
+
+## Step 1: Create a Kubernetes cluster on Amazon EKS
+
+Follow the first two steps in [this AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#)
+to: (1) Create your Amazon EKS cluster (2) Configure your computer to communicate with your cluster.
+
+## Step 2: Create node groups for the Amazon EKS cluster
+
+You can follow "Step 3: Create nodes" in [this AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#) to create node groups. The following section provides more detailed information.
+
+### Create a CPU node group
+
+Typically, we do not run GPU workloads on the Ray head. Create a CPU node group for all Pods except Ray GPU 
+workers, such as the KubeRay operator, Ray head, and CoreDNS Pods.
+
+* A common configuration working for most KubeRay examples:
+  * Instance type: [**m5.xlarge**](https://aws.amazon.com/ec2/instance-types/m5/) (4 vCPU; 16 GB RAM)
+  * Disk size: 256 GB
+  * Desired size: 1, Min size: 0, Max size: 1
+
+### Create a GPU node group
+
+Create a GPU node group for Ray GPU workers.
+
+1. A common configuration working for most KubeRay examples:
+   * AMI type: Bottlerocket NVIDIA (BOTTLEROCKET_x86_64_NVIDIA)
+   * Instance type: [**g5.xlarge**](https://aws.amazon.com/ec2/instance-types/g5/) (1 GPU; 24 GB GPU Memory; 4 vCPUs; 16 GB RAM)
+   * Disk size: 1024 GB
+   * Desired size: 1, Min size: 0, Max size: 1
+
+2. **Please follow Step 4 to install the NVIDIA device plugin.**
+   * If you use `AMI type: Bottlerocket NVIDIA`, there is no need to install NVIDIA device plugin.
+   * For other AMI types, you may need to install the NVIDIA device plugin DaemonSet in order to run GPU-enabled containers in your Amazon EKS cluster.
+   If the GPU nodes have taints, you need to add `tolerations` to `nvidia-device-plugin.yml` to enable the DaemonSet to schedule Pods on the GPU nodes."
+
+3. Add a Kubernetes taint to prevent scheduling CPU Pods on this GPU node group. For KubeRay examples, we add the following taint to the GPU nodes: `Key: ray.io/node-type, Value: worker, Effect: NoSchedule`, and include the corresponding `tolerations` for GPU Ray worker Pods.
```

Comment:
```suggestion
3. Add a Kubernetes taint to prevent scheduling CPU Pods on this GPU node group. For KubeRay examples, add the following taint to the GPU nodes: `Key: ray.io/node-type, Value: worker, Effect: NoSchedule`, and include the corresponding `tolerations` for GPU Ray worker Pods.
```